fenix/app/pings.yaml

```diff
@@ -62,3 +62,15 @@ spoc:
     - https://github.com/mozilla-mobile/fenix/pull/27550#issuecomment-1295027631
   notification_emails:
     - android-probes@mozilla.com
+
+client-deduplication:
```

Comment:
This ping does not define 'reasons'. If we want a fair comparison to the baseline ping, we might want to report the same reasons as well, as much as possible (e.g. we won't be able to do dirty-startup, but we should be fine with the others). We should at least have [active and inactive](https://searchfox.org/glean/rev/f97fc351d54836108c9e3364934e6faa9004550b/glean-core/src/core/mod.rs#874-908)